cli/cmd/inject.go

```diff
@@ -26,23 +29,28 @@ const (
 )
 
 type injectOptions struct {
+	ignoreCluster, disableIdentity bool
```

Comment:
Per comment in install.go, let's move `ignoreCluster` to `proxyConfigOptions`.

Reply:
The issue is that we can't have the proxyConfigOptions.validate() validate this field -- it has to be done in injectOptions.validate() and installOptions.validate() independently, so it seemed better to keep that logic common to those structs. I don't really care that much though if you think it's important.

Reply:
I see the value in combining the flag configuration, though...